### ISSUE TITLE
Please bump jsonwebtoken version to fix vulnerability

### ISSUE THREAD

**Issue by vsly-ar** [arendajaelu/nestjs-passport-apple#7]
Please bump `jsonwebtoken` version from 8.5.1 to 9.0.x to fix this vulnerability: https://github.com/advisories/GHSA-qwph-4952-7xr6

**Comment by arendajaelu** [comment]
@vsly-ru thank you for your comment, did you try to migrate it to 9.0.x and test it?

**Comment by vsly-ar** [comment]
@arendajaelu no, I forked it and tried to bump it myself, but it didn't help much to fix the output of `npm audit`

**Comment by shahabazkc** [comment]
@arendajaelu Please publish the latest code to npm

**Comment by arendajaelu** [comment]
@shahabazkc done ! thank you for the contribution.
